WTWSMS/history/titles/b_asti.txt

```diff
@@ -0,0 +1,3 @@
+589.1.1={
+	holder = 190180 #Gundualdus
+}
```

Comment:
Why the barony?

Reply:
I wanted to give them c_monferrato, but the county is already owned by another Lombard duchy (d_monferrato). d_asti holding b_asti seems the best solution in terms of representing both duchies at once.

Reply:
b_asti is a city, as per the historical situation.

Reply:
I see two solutions:
- let Asti have only b_asti and be a republic
- activate one of the inactive castle-type baronies of c_monferrato

Reply:
I added this character mostly for historical accuracy. Asti being a county-less republic wouldn't block the player from playing as feudal Monferrato.

Reply:
Sorry, but I don't get you, could you please explain further? How would it be historically accurate to create a land-locked republic lead by a historical feudal dynasty?

Reply:
It's historically accurate in terms of the duchy of Asti existing. For "feudal" government, I can give it one castle-type barony.

Reply:
Then the issue is that we would make the county more powerful. I don't know if that is adequate during this period.

Reply:
I don't know either.
I have found that Tortona (b_tortona) remained quite significant after the fall of the WRE. Even Theodoric the Great called it the granary of Liguria. I guess we could activate it.